x/feegrant/codec.go

```diff
@@ -23,3 +37,23 @@ func RegisterInterfaces(registry types.InterfaceRegistry) {
 
 	msgservice.RegisterMsgServiceDesc(registry, &_Msg_serviceDesc)
 }
+
+var (
+	amino = codec.NewLegacyAmino()
+
+	// ModuleCdc references the global x/feegrant module codec. Note, the codec should
+	// ONLY be used in certain instances of tests and for JSON encoding as Amino is
+	// still used for that purpose.
+	//
+	// The actual codec used for serialization should be provided to x/feegrant and
+	// defined at the application level.
+	ModuleCdc = codec.NewAminoCodec(amino)
+)
```

Comment:
```suggestion
```

Is that correct? Come to think about it, we can remove ModuleCdc globals from _all_ modules, right? (we can do in a separate PR, so that this one focused on authz/feegrant)